modules/install-sno-generate-the-discovery-iso.adoc

```diff
@@ -0,0 +1,51 @@
+// This is included in the following assemblies:
+//
+// installing_sno/install-sno-installing-sno.adoc
+
+[id="generate-the-discovery-iso_{context}"]
+
+= Generate the discovery ISO
+
+Single node OpenShift (SNO) installs with a discovery ISO, which the Assisted Installer (AI) generates with the cluster name, base domain, SSH public key, and pull secret.
```

Comment:
Is "base domain" the domain name?

Sugg: Instead of "SSH public key", consider "Secure Shell public key".  That sugg is based on ISG, around p397.